<launch>
  <include file="$(find jetyak_uav_utils)/launch/jetyak_mavros.launch"/>
  <group ns="jetyak_uav_utils">
    <node name="uav_controller" pkg="jetyak_uav_utils" type="dji_pilot_node" output="screen" >
  		<param name="isM100" type="bool" value="true"/>
      <param name="hVelocityMaxBody" type="double" value="3.0"/>
      <param name="hVelocityMaxGround" type="double" value="3.0"/>
      <param name="vVelocityMaxBody" type="double" value="1.0"/>
      <param name="vVelocityMaxGround" type="double" value="1.0"/>
  	</node>
    
    <node name="uav_behaviors" pkg="jetyak_uav_utils" type="behaviors_node" output="screen">
      <rosparam command="load" file="$(find jetyak_uav_utils)/cfg/behaviors.yaml" />
    </node>
    <node name="rc_interpreter" pkg="jetyak_uav_utils" type="rc_interpreter.py" output="screen"/>


  </group>

	<!-- Start capturing the flight data -->
	<!--include file="$(find jetyak_uav_utils)/launch/flight_data.launch"/-->
</launch>
